admin_manual/enterprise/external_storage/s3_swift_as_primary_object_store_configuration.rst

```diff
@@ -1,159 +1,167 @@
-=============================================================
-Configuring S3 and OpenStack Swift Objects as Primary Storage
-=============================================================
+=================================
+Configuring S3 as Primary Storage
+=================================
 
-In ownCloud Enterprise edition, you can configure S3 objects as primary 
-storage. This replaces the default ownCloud ``owncloud/data`` directory. You may 
-still need to keep the ``owncloud/data`` directory for these reasons:
+In ownCloud Enterprise edition administrators can configure Amazon S3 objects as the primary storage.
+Doing this replaces the default ownCloud ``owncloud/data`` directory.
+
+You may still need to keep the ``owncloud/data`` directory for these reasons:
 
 * The ownCloud log file is saved in the data directory
 * Legacy apps may not support using anything but the ``owncloud/data`` directory
 
-You can move your logfile by changing its location in ``config.php``. You may still need 
-``owncloud/data`` for backwards compatibility with some apps.
+Administrators can move their ownCloud log file, by changing its location in ``config.php``.
```

Comment:
Would´nt it be better to „soft push“ admins to move the log file to a different location? It does not make a lot of sense to log to an „internet“ location. What happens if the connection breaks? You can not log anymore.
Something like that it is highly recommended or so...

Reply:
It may well be. I've just reworked the sentence to make use of the active tense, instead of passive.

Reply:
I am still not satisfied. „Can“ is like „nothing happens if I am not doing it“...
I propose to say: It is recommended to move the ownCloud logfile to a different location than the default in owncloud/data by changing its location in config.php.
My concern is, that if you do not give a clear advice and something happens, you get as company a bad mark. If someone does not follow the advice - not your mistake...

Reply:
I see what you mean, now that you've gone into it in greater detail. We can only encourage/push people so far, and the onus is on us only for so much, but I see what you mean.